doc/source/configuration.rst

```diff
@@ -150,6 +152,8 @@ The Nextflow configuration file is used to configure various settings for the *k
 
     * **enable** enables or disables the use of the respective container
     * **cacheDir** the directory where remote the Singularity image from Docker Hub is stored. When using a computing cluster it must be a shared folder accessible to all computing nodes.
+    * **autoMounts** automatically mounts host paths in the executed container and allows the user to run the pipeline from any directory in a cluster [Default: false]. It requires the `*user bind control* <https://sylabs.io/guides/3.7/admin-guide/configfiles.html?highlight=user%20bind%20control#bind-mount-management`>__ feature activated in Singularity installation [Default: "YES"]
```

Comment:
```suggestion
    * **autoMounts** automatically mounts host paths in the executed container and allows the user to run the pipeline from any directory in a cluster [Default: false]. It requires the `*user bind control* <https://sylabs.io/guides/3.7/admin-guide/configfiles.html?highlight=user%20bind%20control#bind-mount-management`>__ feature activated in Singularity installation [Default: true]
```

Reply:
In Singularity documentation they say that the default option is YES, that's why I reported it as it is although it doesn't really fit with our style :)

Reply:
Ah sorry, I meant to change the "autoMounts" default to true!

Reply:
It is indeed a bit confusing... The first "Default" is related to autoMounts and the second to "user bind control". I'll make it clearer. Plus, the first Default is set to false because that's the default autoMounts setting in Nextflow and I though to provide that one and not "our" default, which is "true".

Reply:
Hmm, we are usually documenting the defaults we set in configuration.rst, so I think we should do the same here. Why document the default in Nextflow, not sure that would be clear.